dataservice/extensions/flask_indexd.py

```diff
@@ -166,20 +178,34 @@ def update(self, record):
             del req_body['size']
             del req_body['hashes']
 
-        # If acl changed, update all previous version with new acl
+        # Updates to the acl have been disabled in favor of the authz field
         if record.acl != old['acl']:
```

Comment:
Will `acl` always be present in the response or are they planning to remove it completely at some point? If the latter, should we change to `old.get('acl')`

Reply:
Also won't `record.acl` always be None now (since `acl_deprecation` validation would have raised an exception if it wasn't)?